metrics/internal/collectors/object-bucket.go

```diff
@@ -143,6 +144,14 @@ func (c *ObjectBucketCollector) getAllObjectBuckets(cephObjectStore cephv1.CephO
 		LabelSelector: selector,
 	}
 
+	// collect the OBC list
+	objectBucketClaimList, err := c.bktclient.ObjectbucketV1alpha1().
+		ObjectBucketClaims(cephObjectStore.Namespace).List(context.TODO(), listOpts)
+	// any error, don't panic
+	if err != nil {
+		// make a non-nil OBC List with Items 'nil'
+		objectBucketClaimList = &libbucket.ObjectBucketClaimList{Items: nil}
+	}
```

Comment:
Why not just initialize `objectBucketClaimList` with zero-value and then do the List operation?

Reply:
What if the `List()` function returns a `nil, err` value (at line#148)?
Even if we had initialized the `objectBucketClaimList` with a zero value (before the line#148), it would be reset to a `nil` value.
So it is always safer to reset it to a zero (non-nil) value on error.